examples/event-driven-schedule.yaml

```diff
@@ -0,0 +1,24 @@
+document:
+  dsl: 1.0.0-alpha1
+  namespace: examples
+  name: event-driven-schedule
+  version: 1.0.0-alpha1
+schedule:
+  on:
+    one:
+      with:
+        type: com.example.hospital.events.patients.heartbeat.low
+do:
```

Comment:
Don't we have to use a listener task instead in this example?

Reply:
Depends on what you want to achieve.

The objective here is to trigger a new workflow when and only when the event is received.

Using a listen task would mean you'd have started a workflow out of the blue, then wait potentially forever for an event to be consumed.

Reply:
Based on our private conversation, we agreed to follow up with an issue to review the `schedule.on` naming since it may confuse users. Some may expect that `schedule` enforces a timestamp definition.